android-components/components/feature/search/src/main/assets/search/list.json

```diff
@@ -126,7 +126,7 @@
     "ca": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "ddg", "wikipedia-ca"
```

Comment:
In your [spreadsheet](https://docs.google.com/spreadsheets/d/1Vt-VWMypfadHGjey6KW109sfIQ9LHzq-QqX9EATDTkk/edit#gid=0), you reported this as `['Google','Bing' 'Duck Duck Go', 'Viquipèdia (ca)']`, but we don't have `bing` here. Is this something we also need to add? I think I know the answer here which is that we asked the search team to determine what we should add, but just looking to verify.

I am gonna ask where we got the desktop search engine order from just to double check.

Reply:
Eagle eye! I erroneously added `bing` in the spreadsheet to "`ca`". It is not there and I am asking search team in bugzilla to answer if we should.

Reply:
We need to add "bing" to ca as added by @Standard8 in the latest bugzilla comment.